openreview/venue/webfield/paperBidWebfield.js

```diff
@@ -1,6 +1,17 @@
 // Webfield component
 const committeeName = entity.content.committee_name?.value
-const affinityScoreId = domain.content[`${committeeName.toLowerCase()}_affinity_score_id`]?.value
+const reviewersName = domain.content['reviewers_name']?.value
```

Comment:
Thanks for doing this, I think there will be other places with the same issue:

- assignment_pre_process.js
- invite_assignment_pre_process.js
- proposed_assignment_pre_process.js


You can fix them in this PR or make a new one. But I would like to reproduce the issue with a test first.

Reply:
okay! I think I should make these changes in another PR, I'll add tests for each one